modules/oe_webtools_captcha/config/install/oe_webtools_captcha.settings.yml

```diff
@@ -1 +1 @@
-validationEndpoint: 'https://europa.eu/webtools/rest/captcha/verify'
+validationEndpoint: 'https://webtools.europa.eu/rest/captcha/verify'
```

Comment:
For information, please check https://webgate.ec.europa.eu/fpfis/wikis/pages/viewpage.action?pageId=1851981836 especially **§ Communication sent on April 14th, 2024**  for possible impacts.

Reply:
This can be overridden by an environent variable.